### ISSUE TITLE
Quantum/Production Source maps

### ISSUE THREAD

**Issue by nchanged** [fuse-box/fuse-box#1375]
Yes, it's happening in 3.6 ;-)

**Comment by misantronic** [comment]
that's great news!!

**Comment by jpike88** [comment]
@nchanged what! how?

**Comment by nchanged** [comment]
@jpike88 @misantronic Indeed. Well, i've cooked a really magical library https://github.com/fuse-box/sourcemap-blender
And in fact, integrated it in a branch, it's working flawlessly with Quantum. Only on project files tho (not vendor - but who cares). I will post the details later and ask you guys to test it out, and give your ideas / suggestions.

**Comment by guillaume86** [comment]
Congrats that sounds awesome! Any target date for 3.6 release (or just integration into the "next" releases)?
Can't wait to update my build config :).

**Comment by nchanged** [comment]
It's working for react projects , but there are cases where it does not. I will make a release of 3.6 without the sourcemap support. 3.7 next will have it though

**Comment by RXminuS** [comment]
So just to be clear, this is currently not available in 3.6 or it is available just not enabled?

**Comment by guillaume86** [comment]
> So just to be clear, this is currently not available in 3.6 or it is available just not enabled?

Not available

**Comment by nchanged** [comment]
I will publish a next version this week  which will contain this feature and let you guys know

**Comment by nchanged** [comment]
Hey guys, Give it a try `fuse-box@3.7.0-next.6`

* Make sure you have cache disabled and `sourceMaps: true`
* Make sure that the source maps are reachable, (check the output) If it's wrong - do the following

```
 QuantumPlugin({
            sourceMaps: {
                path: "/",
            },
        }),
```

Add `vendor : true` if you want to see the vendor Source maps (however it will take ages to generate and all I could do it just map the file names - so it won't be  "true" VENDOR sourcemaps - but it should be helpful enough.

So far, I've managed to see my working sourcemaps on my heavy project (+1000 files) and it did work as expected. The PR is [here](https://github.com/fuse-box/fuse-box/pull/1431) I will merge it once you guys confirm that it's working for you and also after we've ironed out the details and options.

**Comment by misantronic** [comment]
I just tested it in my production system and it works without any problems! thanks a million!! <3

**Comment by RXminuS** [comment]
Also tested in production...seems to work great for me too! Awesome job, can't tell you how much of a life save this is in Sentry

**Comment by RXminuS** [comment]
It doesn't seem to work on `fuse-box@3.7.0-next.8` anymore. `Object literal may only specify known properties, and 'sourceMaps' does not exist in type 'IQuantumExtensionParams'`

**Comment by nchanged** [comment]
@RXminuS it's only in `fuse-box@3.7.0-next.6` as mentioned above

**Comment by nchanged** [comment]
Hey guys, the branch has been finally merged! `fuse-box@3.7.0-next.10`
